lib/ama/styles/version.rb

```diff
@@ -2,6 +2,6 @@
 
 module AMA
   module Styles
-    VERSION = '2.15.2'
+    VERSION = '2.15.3'
```

Comment:
Can you bump the minor version instead since it's not a bug fix? (2.16.0)

Reply:
Done.